tests/http/bad-actors.test.js

```diff
@@ -1,16 +0,0 @@
-var server = require('../../server');
```

Comment:
I talked to Jbuck, and he created this test to test that urls with non ascii characters sent us to a 400. However, we now redirect to the home for urls that are not in react-router, so I saw no more need to test that we were doing something we are not.

I couldn't figure out what else this test was trying to do beyond that, and there wasn't anything else in the test saying why a url couldn't contain non ascii characters, so I actually consider it a pretty toxic test.